### PR TITLE
Added start-at and tags commands

### PR DESCRIPTION
**Pull request by giper45**

### CODE REVIEW

File: pages/common/ansible-playbook.md [hunk 1]
```diff
@@ -18,3 +18,11 @@
 - Run tasks in playbook with extra variables defined in a json file:
 
 `ansible-playbook {{playbook}} -e "@{{variables.json}}"`
+
+- Run tasks in playbook for a given tags:
```
Comment (zdroid) [inline pages/common/ansible-playbook.md:22]:
```suggestion
- Run tasks in playbook for the given tags:
```

File: pages/common/ansible-playbook.md [hunk 2]
```diff
@@ -18,3 +18,11 @@
...
+
+`ansible-playbook {{playbook}} --tags {tags}}`
+
+- Run tasks in playbook starting by task_name:
```
Comment (zdroid) [inline pages/common/ansible-playbook.md:26]:
```suggestion
- Run tasks in a playbook starting at a specific task:
```

File: pages/common/ansible-playbook.md [hunk 3]
```diff
@@ -18,3 +18,11 @@
...
 
 `ansible-playbook {{playbook}} -e "@{{variables.json}}"`
+
+- Run tasks in playbook for the given tags:
+
+`ansible-playbook {{playbook}} --tags {tags}}`
+
+- Run tasks in a playbook starting at a specific task:
+
+`ansible-platybook {{playbook}} --start-at {{task_name}}`
```
Comment (sbrl) [inline pages/common/ansible-playbook.md:28]: Typo here?
```suggestion
`ansible-platbook {{playbook}} --start-at {{task_name}}`
```

Comment (giper45) [reply pages/common/ansible-playbook.md:28]: Typo, it should be
ansible-playbook

Comment (sbrl) [reply pages/common/ansible-playbook.md:28]: Oops lol, suggestion updated

**Pull request merged**